apis/v1alpha1/httproute_types.go

```diff
@@ -107,19 +107,6 @@ type HTTPRouteSpec struct {
 	// +kubebuilder:validation:MinItems=1
 	// +kubebuilder:validation:MaxItems=16
 	Rules []HTTPRouteRule `json:"rules"`
-
-	// ExtensionRef is an optional, implementation-specific extension to the
-	// "host" block. The resource may be "configmaps"  or an implementation-defined
-	// resource (for example, resource "myroutehosts" in group "networking.acme.io").
-	//
-	// If the referent cannot be found,
-	// the GatewayClass's "InvalidParameters" status condition
-	// will be true.
-	//
-	// Support: custom
-	//
-	// +optional
-	ExtensionRef *RouteHostExtensionObjectReference `json:"extensionRef,omitempty"`
```

Comment:
This extension point seems extraneous and is not present in other route types. Should implementers choose to go for a specific Route type if this is needed?

Reply:
I completely agree with removal of this one. As discussed last week, anyone wanting to add extension points that were 1:1 with the route like this could use annotations or name-mapping.